prime-angular-frontend/src/app/modules/site-registration/shared/guards/site.guard.ts

```diff
@@ -40,9 +38,67 @@ export class SiteGuard extends BaseGuard {
           // Store the site for access throughout creation and updating of a
           // site, which will allows provide the most up-to-date site
           this.siteService.site = site;
-          // TODO always resolve until routes are lock down
-          return true;
+
+          return this.routeDestination(routePath, site);
         })
       );
   }
+
+  /**
+   * @description
+   * Determine the route destination based on the site status.
+   */
+  private routeDestination(routePath: string, site: Site) {
+    if (site) {
+      return (site.submittedDate)
+        ? this.manageSubmittedSiteRouting(routePath, site)
+        : true;
+    }
+
+    // Otherwise, prevent the route from resolving
+    return false;
+  }
+
+  private manageSubmittedSiteRouting(routePath: string, site: Site) {
+    return this.manageRouting(routePath, SiteRoutes.SITE_MANAGEMENT, site);
+  }
+
+  private manageRouting(routePath: string, defaultRoute: string, site: Site): boolean {
+
+    const childRoute = routePath.includes('?')
```

Comment:
You actually don't need to check for the `?` if you use `.split('?')` and it doesn't exist you'll have an array with a single index, which when you chain it with `.shift()` results in you have the same thing you started with so can one-liner it with no ternary